### PR TITLE
Remove sctp-transport

### PR DESCRIPTION
**Pull request by henbos**
One of the steps needed for #621.

> AFTER THIS PR HAS BEEN APPROVED BUT BEFORE LANDING IT:
> Create a corresponding webrtc-provisional-stats PR adding these metrics to that document instead.

***
<a href="https://pr-preview.s3.amazonaws.com/henbos/webrtc-stats/pull/627.html" title="Last updated on Jun 9, 2022, 1:40 PM UTC (29695b2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/webrtc-stats/627/af5fb42...henbos:29695b2.html" title="Last updated on Jun 9, 2022, 1:40 PM UTC (29695b2)">Diff</a>

### CODE REVIEW

**Comment by Orphis** [comment]
I believe those would still be useful and could be done in Chrome with the new SCTP implementation.

**Comment by henbos** [comment]
Are you interested in implementing it any time soon? (If not, we can always add them back to the spec at a later date if removed here)

File: webrtc-stats.html [hunk 1]
```diff
@@ -3065,151 +3054,58 @@ <h2>
...
-            <section>
-              <h3>
-                <dfn>RTCDtlsRole</dfn> enum
-              </h3>
-              <div>
-                <pre class="idl">enum RTCDtlsRole {
-        "client",
-        "server",
-        "unknown",
+          <section>
```
Comment (henbos) [inline webrtc-stats.html:3057]: (Drive-by indentation fixes)
